packages/kit/src/runtime/server/index.js

```diff
@@ -32,12 +33,15 @@ export async function respond(incoming, options, state = {}) {
 	}
 
 	const headers = lowercase_keys(incoming.headers);
+	const nonce = randomBytes(32).toString('base64');
```

Comment:
Is a plain base64 nonce safe to use everywhere or does it need to replace some chars to avoid escaping a context its used in (http attribute, http header value, ???)

Reply:
Base64 should be safe here. In the linked file they replace `+`, `/` and `=`, none of which are problematic in this context. In the Vite plugin this is done because the result is used as filenames, where `/` in particular is problematic.

Looking at [the list of base64 symbols](https://base64.guru/learn/base64-characters) everything is safe for this context.